text/0118-search-api.md

```diff
@@ -468,6 +523,8 @@ Whether `nbHits` is exhaustive.
 
 Added to the search response when `facetsDistribution` is set for a search query. It contains the number of remaining candidates for each specified facet in the `facetsDistribution` search parameter.
 
+If a field distributed as a facet contains no value, it is returned as a `facetDistribution` field with an empty object as value.
```

Comment:
What do you mean with this?

Reply:
Trying to explain the expected behavior here https://github.com/meilisearch/meilisearch/issues/2352#issue-1216056948  !

Reply:
I will rephrase it soon to be more precise with our last tests

Reply:
Ah I see!